quick-start-with-tidb.md

```diff
@@ -6,25 +6,125 @@ aliases: ['/docs/dev/quick-start-with-tidb/','/docs/dev/test-deployment-using-do
+
+1. Download and install TiUP:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
+    ```
+
+2. Declare the global environment variable:
+
+    > **Note:**
+    >
+    > After the installation, TiUP displays the absolute path of the corresponding `profile` file. You need to modify the following `source` command according to the path.
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    source .bash_profile
+    ```
+
+3. Start the cluster in the current session:
+
+    - If you want to start a TiDB cluster of the latest version with 1 TiDB instance, 1 TiKV instance, 1 PD instance, and 1 TiFlash instance, run the following command:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        tiup playground
+        ```
+
+    - If you want to specify the TiDB version and the number of the instances of each component, run a command like this:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        tiup playground v5.0.1 --db 2 --pd 3 --kv 3 --monitor
+        ```
+
+        The command downloads a version cluster to the local machine and starts it, such as v5.0.1. `--monitor` means that the monitoring component is also deployed.
+
+        To view the latest version, run `tiup list tidb`.
+
+        This command returns the access methods of the cluster:
+
+        ```log
+        CLUSTER START SUCCESSFULLY, Enjoy it ^-^
+        To connect TiDB: mysql --host 127.0.0.1 --port 4000 -u root
+        To connect TiDB: mysql --host 127.0.0.1 --port 4001 -u root
+        To view the dashboard: http://127.0.0.1:2379/dashboard
+        To view the monitor: http://127.0.0.1:9090
+        ```
+
+        > **Note:**
+        >
+        > For the playground operated in this way, after the test is finished, TiUP will clean up the original cluster data. You will get a new cluster after re-running the command.
+        > If you want the data to be persisted on storage，run `tiup --tag <your-tag> playground ...`. For details, refer to [TiUP Reference Guide](/tiup/tiup-reference.md#-t---tag).
+
+4. Start a new session to access TiDB:
 
-This document describes how to try out TiDB, a distributed SQL database, in a few minutes. You can choose any of the following two methods to get started with the TiDB database platform:
+    + Use the TiUP client to connect to TiDBs.
 
-- [Deploy a local test environment using TiUP Playground](#deploy-a-local-test-environment-using-tiup-playground)
-- [Set up a test environment on a single machine using TiUP Cluster](#set-up-a-test-environment-on-a-single-machine-using-tiup-cluster)
+        {{< copyable "shell-regular" >}}
 
-## Deploy a local test environment using TiUP Playground
+        ```shell
+        tiup client
```

Comment:
```
my:root@127.0.0.1:4000=> \conninfo
Connected with driver mysql (root@tcp(127.0.0.1:4000)/?loc=Local&parseTime=true&sql_mode=ansi)
my:root@127.0.0.1:4000=> select "foo";
error: mysql: 1054: Unknown column 'foo' in 'field list'
my:root@127.0.0.1:4000=> select 'foo';
 foo 
-----
 foo 
(1 row)

my:root@127.0.0.1:4000=> select 'foo' || 'bar';
 'foo' || 'bar' 
----------------
 foobar         
(1 row)

my:root@127.0.0.1:4000=>  
```
People probably don't expect an SQL mode that is different than what MySQL (and TiDB) does by default.

Reply:
This PR aims to solve some specific issues and is large enough for reviewers. To merge this PR quickly, I suggest we keep each PR relatively small. 

For new ideas for improving this page, may I suggest that we discuss and apply them in another new PR? I added a "requires-followup" label to follow up on your suggestions after we merge this PR.

Reply:
That looks like another issue with usql :(

The goal here was to remove a dependency, but it's got some issues. We'll have to ask the tiup team to fix this.

Reply:
Looks like usql is doing this on purpose. Probably to keep the behaviour similar for various DB servers.

https://github.com/xo/usql/blob/ac3c08f3d9cd7e92712479d3898a3b77f08a2c5e/drivers/mysql/mysql.go#L40-L44